main/repl/timerServices.md

```diff
@@ -77,8 +77,8 @@ history[32] 1632170321864n // about 3 seconds later, in milliseconds since epoch
 - Returns: `{ Notifier<Timestamp> }`
 
 Creates and returns a `Notifier` object. It repeatedly delivers updates at times
-that are a multiple of the passed-in `interval` value, with the first update happening
-the value of `delay` after the notifier is created.
+that are a multiple of the provided `interval` value, with the first update happening
+after the provided `delay` value.
```

Comment:
This fails to define or link to the semantics of RelativeTime (units, shape, etc.), and therefore of `delay` and `interval`. But it should do so or else the documentation is of little value. So, what _is_ a relative time and how is it expressed?